pyproject.toml

```diff
@@ -201,16 +201,22 @@ dev = [
   "python-multipart",
   "psutil",
   # Linting
-  "pylint==2.15.10",
-  # Code formatting
-  "black[jupyter]==22.6.0",
+  "pylint",
+  "farm-haystack[formatting]",
```

Comment:
Oh cool this works? I would have thought this may caused some sort of circular dependency XD

Reply:
Thanks for the explanation.